### ISSUE TITLE
HomePilot Gateway premium 15001001

### ISSUE THREAD

**Issue by Homeworker11** [homecineplexx/ioBroker.homepilot20#62]
Bitte die Neue "HomePilot Gateway premium 15001001" Zentrale zu den unterstützten Geräten hinzufügen.

**Comment by Haui74** [comment]
Wäre auch dafür :)

**Comment by homecineplexx** [comment]
> Bitte die Neue "HomePilot Gateway premium 15001001" Zentrale zu den unterstützten Geräten hinzufügen.

Hallo
Funktioniert dieses Gateway? 
Als Bridge oder ganz normal?

danke

**Comment by Haui74** [comment]
Der Adapter wird grün, aber im Log sind nur Fehlermeldungen


<html>
<body>

2023-12-20 21:29:02.748 - warn: homepilot20.0 (2297827) Transmitter sensors -> Cannot connect to Homepilot: {"statusCode":401,"body":"Unauthorized","headers":{"server":"nginx/1.22.1","date":"Wed, 20 Dec 2023 20:29:02 GMT","content-type":"text/html;charset=utf-8","transfer-encoding":"chunked","connection":"keep-alive"},"request":{"uri":{"protocol":"http:","slashes":true,"auth":null,"host":"192.168.0.169:80","port":"80","hostname":"192.168.0.169","hash":null,"search":"?devtype=Transmitter","query":"devtype=Transmitter","pathname":"/v4/devices","path":"/v4/devices?devtype=Transmitter","href":"http://192.168.0.169:80/v4/devices?devtype=Transmitter"},"method":"GET","headers":{"0":{},"1":{"Content-Type":"application/json"}}}}
--
2023-12-20 21:29:02.751 - warn: homepilot20.0 (2297827) Read sensors -> Cannot connect to Homepilot: {"statusCode":401,"body":"Unauthorized","headers":{"server":"nginx/1.22.1","date":"Wed, 20 Dec 2023 20:29:02 GMT","content-type":"text/html;charset=utf-8","transfer-encoding":"chunked","connection":"keep-alive"},"request":{"uri":{"protocol":"http:","slashes":true,"auth":null,"host":"192.168.0.169:80","port":"80","hostname":"192.168.0.169","hash":null,"search":"?devtype=Sensor","query":"devtype=Sensor","pathname":"/v4/devices","path":"/v4/devices?devtype=Sensor","href":"http://192.168.0.169:80/v4/devices?devtype=Sensor"},"method":"GET","headers":{"0":{},"1":{"Content-Type":"application/json"}}}}


</body>
</html>

**Comment by homecineplexx** [comment]
hmm...das problem ist, dass es bei dem normalen gateway, das ich habe auch transmitter und sensor gibt und die werden eben auch regelmäßig abgefragt. 
jetzt hab ich allerdings dein gateway nicht und ist schwierig dafür etwas einzubauen, aber ich schau mir das an.

kannst du mir vielleicht mal ein foto zeigen wie deine oberfläche aussieht, wenn du über den browser einsteigst?

**Comment by Haui74** [comment]
Die Konfiguration funktioniert ausschließlich über die App. Das Gateway besitzt keine Weboberfläche. :(

**Comment by homecineplexx** [comment]
> Die Konfiguration funktioniert ausschließlich über die App. Das Gateway besitzt keine Weboberfläche. :(

wirklich nicht? bist dir sicher? 
ok, aber das gateway bekommt eine ip zugewiesen richtig???

**Comment by Homeworker11** [comment]
Hallo;

Das neue Gateway besitzt wirklich keine Weboberfläche und es gibt wahrscheinlich auch keine API dafür.
Also am besten beim Hersteller beschweren, und das Teil gegen das Original Rademacher  Gateway Tauschen.

**Comment by Haui74** [comment]
> > Die Konfiguration funktioniert ausschließlich über die App. Das Gateway besitzt keine Weboberfläche. :(
> 
> wirklich nicht? bist dir sicher? ok, aber das gateway bekommt eine ip zugewiesen richtig???
> 
> wie schaut denn deine app aus?

Ja, das Gateway hat eine IP. Die App heißt "Homepilot". Es gibt auch einen Alexa Skill dafür. Du kannst dir gerne mal die App ionstallieren und bekommst dann meine Zugangsdaten, wenn du möchtest.

**Comment by homecineplexx** [comment]
> > > Die Konfiguration funktioniert ausschließlich über die App. Das Gateway besitzt keine Weboberfläche. :(
> > 
> > 
> > wirklich nicht? bist dir sicher? ok, aber das gateway bekommt eine ip zugewiesen richtig???
> > wie schaut denn deine app aus?
> 
> Ja, das Gateway hat eine IP. Die App heißt "Homepilot". Es gibt auch einen Alexa Skill dafür. Du kannst dir gerne mal die App installieren und bekommst dann meine Zugangsdaten, wenn du möchtest.


ist das diese app?

![homepilot](https://github.com/homecineplexx/ioBroker.homepilot20/assets/43374575/18764098-4e3a-4256-8eaa-b6a67d794fc1)

**Comment by Haui74** [comment]
ja, isse

**Comment by Haui74** [comment]
Schreib mich mal per E Mail an, dann geb ich dir die Zugangsdaten: haui74@unitybox.de

**Comment by pda1103** [comment]
moinsen homecineplexx

gibt es schon Neuigkeiten bzgl. der Einbindung vom Homepilot Gateway premium?

**Comment by homecineplexx** [comment]
leider nein...mein kontakt den ich bei homepilot habe (hatte), meldet sich diesbezüglich leider nicht

**Comment by AdamK-DE** [comment]
hier leider das selbe :( kaufte mir extra dieses starter-set mit 2x motoren + gateway premium da mir das als durch deinen adapter supported in der ioB FB Gruppe genannt wurde aber ist nicht :( nur fehler.. 

eine API scheint es da aber schon zu geben. http läuft ja auch wenns keine weboberfläche mehr gibt...

dein request: http://192.168.178.117/v4/devices?devtype=Transmitter
ergibt->

{"error_description":"GET URI not found","error_code":5002}
